code/GUI.py

```diff
@@ -23,8 +23,8 @@
 import plyer
 import os
 import Tracker
-import pystray
-from pystray import MenuItem as item, Menu as menu
+import pystray._win32
```

Comment:
Are we sure this will not break the app for mac?

Reply:
Yes, it might, will fix that in the next commit.